guides/common/modules/con_additional-deployment-tasks.adoc

```diff
@@ -0,0 +1,40 @@
+[id="additional-deployment-tasks_{context}"]
+= Additional deployment tasks
+
+{Project} offers a range of additional capabilities that you can use to further enhance your {Project} deployment.
+For example:
```

Comment:
Without "For example", it might look like this is supposed to be a complete list. Which it isn't, it's just an overview of the additional tasks we want to highlight. So no change for now, please feel free to explain further if I'm missing something :)